### ISSUE TITLE
Critical: an error occurred when `--overview` key request

### ISSUE THREAD

**Issue by Tim55667757** [Tim55667757/TKSBrokerAPI#71]
```
$ tksbrokerapi -v 10 -o --output portfolio.md 

TKSBrokerAPI.py     L:4457 DEBUG   [2022-11-06 22:29:24,421] >>> TKSBrokerAPI module started at: [2022-11-06 19:29:24] UTC, it is [2022-11-06 22:29:24] local time
TKSBrokerAPI.py     L:4471 DEBUG   [2022-11-06 22:29:24,422] TKSBrokerAPI major.minor.build version used: [1.5.dev0]
TKSBrokerAPI.py     L:4472 DEBUG   [2022-11-06 22:29:24,422] Host CPU count: [8]
TKSBrokerAPI.py     L:210  DEBUG   [2022-11-06 22:29:24,422] Bearer token for Tinkoff OpenAPI set up from environment variable `TKS_API_TOKEN`. See https://tinkoff.github.io/investAPI/token/
TKSBrokerAPI.py     L:223  DEBUG   [2022-11-06 22:29:24,422] Main account ID [2000096541] set up from environment variable `TKS_ACCOUNT_ID`
TKSBrokerAPI.py     L:272  DEBUG   [2022-11-06 22:29:24,422] Broker API server: https://invest-public-api.tinkoff.ru/rest
TKSBrokerAPI.py     L:430  DEBUG   [2022-11-06 22:29:24,444] Local cache with raw instruments data is used: [dump.json]
TKSBrokerAPI.py     L:431  DEBUG   [2022-11-06 22:29:24,444] Dump file was last modified [2022-11-06 19:28:12] UTC
TKSBrokerAPI.py     L:1655 DEBUG   [2022-11-06 22:29:24,444] Requesting portfolio of a client. Wait, please...
TKSBrokerAPI.py     L:1506 DEBUG   [2022-11-06 22:29:24,444] Requesting current actual user's portfolio. Wait, please...
TKSBrokerAPI.py     L:1512 DEBUG   [2022-11-06 22:29:24,585] Records about user's portfolio successfully received
TKSBrokerAPI.py     L:1528 DEBUG   [2022-11-06 22:29:24,586] Requesting current open positions in currencies and instruments. Wait, please...
TKSBrokerAPI.py     L:1534 DEBUG   [2022-11-06 22:29:24,698] Records about current open positions successfully received
TKSBrokerAPI.py     L:1550 DEBUG   [2022-11-06 22:29:24,698] Requesting current actual pending orders. Wait, please...
TKSBrokerAPI.py     L:1556 DEBUG   [2022-11-06 22:29:24,804] [0] records about pending orders received
TKSBrokerAPI.py     L:1572 DEBUG   [2022-11-06 22:29:24,804] Requesting current actual stop orders. Wait, please...
TKSBrokerAPI.py     L:1578 DEBUG   [2022-11-06 22:29:24,911] [0] records about stop orders received
TKSBrokerAPI.py     L:4808 DEBUG   [2022-11-06 22:29:24,922] Traceback (most recent call last):
  File "/Users/tim/projects/TKSBrokerAPI/tksbrokerapi/TKSBrokerAPI.py", line 4572, in Main
    server.Overview(show=True, details="full")
  File "/Users/tim/projects/TKSBrokerAPI/tksbrokerapi/TKSBrokerAPI.py", line 2029, in Overview
    view["analytics"]["distrByCurrencies"]["rub"]["cost"] += view["analytics"]["distrByAssets"]["Ruble"]["cost"]
KeyError: 'rub'

TKSBrokerAPI.py     L:4809 DEBUG   [2022-11-06 22:29:24,922] Please, check issues or request a new one at https://github.com/Tim55667757/TKSBrokerAPI/issues
TKSBrokerAPI.py     L:4819 ERROR   [2022-11-06 22:29:24,922] An issue occurred with TKSBrokerAPI module! See full debug log in [TKSBrokerAPI.log] or run TKSBrokerAPI once again with the key `--debug-level 10`. Summary code: 255
TKSBrokerAPI.py     L:4823 DEBUG   [2022-11-06 22:29:24,922] >>> TKSBrokerAPI module work duration: [0:00:00.501383]
TKSBrokerAPI.py     L:4824 DEBUG   [2022-11-06 22:29:24,922] >>> TKSBrokerAPI module finished: [2022-11-06 19:29:24 UTC], it is [2022-11-06 22:29:24] local time

Process finished with exit code 255
```

**Comment by Tim55667757** [comment]
Server API not return current available rubles in currency portfolio distribution.

Fast hack to avoid issues #71 in `Portfolio distribution by currencies` and `Portfolio distribution by countries` sections: adding `rub` currency and `"[RU] Российская Федерация"` before calculate statistics

**Comment by Tim55667757** [comment]
Build: https://app.travis-ci.com/github/Tim55667757/TKSBrokerAPI/builds/257475766

Fixed in 1.4.90 release build: https://pypi.org/project/tksbrokerapi/1.4.90/
